### PR TITLE
removed hidden overflow for schema compare splitview

### LINKED ISSUE #6609: Schema Compare scrolling doesn't allow showing text compare controls
- Azure Data Studio Version: `master`

Steps to Reproduce:

1. Run a schema compare, with on a machine with smaller resolution or with the terminal panel visible

Expected: It's possible to scroll the text compare editors into the visible screen area

Actual: It is not possible to see the text compare editors.  The only visible scrollbar controls the item grid, but no way to bring the text editors into view.

image

### PR DESCRIPTION
**Pull request by ellbosch**
Fixes #6609.

### CODE REVIEW

File: extensions/schema-compare/src/schemaCompareMainWindow.ts [hunk 1]
```diff
@@ -346,7 +346,7 @@ export class SchemaCompareMainWindow {
 		this.resetButtons(ResetButtonState.afterCompareComplete);
 
 		if (this.comparisonResult.differences.length > 0) {
-			this.flexModel.addItem(this.splitView, { CSSStyles: { 'overflow': 'hidden' } });
```
Comment (Charles-Gagnon) [inline extensions/schema-compare/src/schemaCompareMainWindow.ts]: How does this look for very long strings?

**Comment by Charles-Gagnon** [comment]
Is there an issue associated with this change? Let's make sure we have issues tracking all fixes/improvements we're making!

File: extensions/schema-compare/src/schemaCompareMainWindow.ts (hunk 1 above)
Comment (udeeshagautam) [reply extensions/schema-compare/src/schemaCompareMainWindow.ts]: @ellbosch can you please add a snapshot of how SC looks before/after the change

Comment (kisantia) [reply extensions/schema-compare/src/schemaCompareMainWindow.ts]: Yes please add gifs

**Comment by ellbosch** [comment]
Thanks @Charles-Gagnon, my apologies--just updated the description to reflect that this fixes #6609.

File: extensions/schema-compare/src/schemaCompareMainWindow.ts (hunk 1 above)
Comment (ellbosch) [reply extensions/schema-compare/src/schemaCompareMainWindow.ts]: **Old functionality:**
old

**New functionality:**
new

**Comment by ellbosch** [comment]
Thanks @Charles-Gagnon, I didn't know this! Also including another gif to show overflow with the terminal panel visible.
new2

**Comment by udeeshagautam** [comment]
@kburtram, @yualan - is the double scroll bar okay fix for accessibility?

**Comment by udeeshagautam** [comment]
@ellbosch can you please resolve conflicts and complete this. That will give people a way to test new vs old behavior. Let's discuss if there are any concerns.

File: .vscode/launch.json [hunk 2]
```diff
@@ -207,18 +210,25 @@
 			]
 		},
 		{
-			"name": "Search and Renderer processes",
+			"name": "Debug Renderer and search processes",
```
Comment (udeeshagautam) [inline .vscode/launch.json]: Okay I am confused as to why do these show as changes?

**Comment by alanrenmsft** [comment]
@ellbosch do you still need this branch?

**Pull request merged**